addons/mail/static/src/components/message/message.xml

```diff
@@ -85,7 +85,15 @@
                                         (from <a class="o_Message_originThreadLink" t-att-href="message.originThread.url" t-on-click="_onClickOriginThread">#<t t-esc="message.originThread.name"/></a>)
                                     </t>
                                     <t t-else="">
-                                        on <a class="o_Message_originThreadLink" t-att-href="message.originThread.url" t-on-click="_onClickOriginThread"><t t-esc="message.originThread.name"/></a>
+                                        on
+                                        <a class="o_Message_originThreadLink" t-att-href="message.originThread.url" t-on-click="_onClickOriginThread">
+                                            <t t-if="message.originThread.name">
+                                                <t t-esc="message.originThread.name"/>
+                                            </t>
+                                            <t t-else="">
+                                                document
```

Comment:
need to add new string in .pot file

Reply:
This is a first for me. Opening the .pot (raw) file is kind of scary. I know there is software to manage them but I don't know if that's the way you do it.

Reply:
You can export everything from the admin debug setting interface, but usually for bugfix you just copy and adapt an existing occurrence in the pot file, taking example from a string in another similar template (or even better in the same template, if there is one already).